test/e2e/fixture-builder.js

```diff
@@ -555,7 +555,7 @@ class FixtureBuilder {
           [toHex(1337)]: [
             {
               address: `__FIXTURE_SUBSTITUTION__CONTRACT${SMART_CONTRACTS.NFTS}`,
-              name: 'TestDappCollectibles',
```

Comment:
All of the Collectibles -> NFTs changes are to adapt to [the term unification change introduced in `@metamask/test-dapp` v7.0.0](https://github.com/MetaMask/test-dapp/commit/57958f0cc25ac1d88d1d21dc312094777b2b8f5a#diff-06572a96a58dc510037d5efa622f9bec8519bc1beab13c9f251e97e657a9d4edR14)